### DISCUSSION TITLE
Formating the Nav Menu

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1705] [Q&A]
How to format the nav menu to:
1- Remove the down arrow after menu-items or replace it with another character such as bar |
2- remove the link title property to disable showing the title when hovering of the menu-items

**Comment by bacoords** [comment]
1- Take a look at the [enable-caret sass option](https://getbootstrap.com/docs/5.1/customize/options/)
2- You'll probably need to override the bootstrap navwalker if you want to globally remove the title property. On the other hand, you can actually customize the 'title' in each menu item by using the ["Screen Options" tab on the Menus page](https://wordpress.org/support/article/appearance-menus-screen/)

**Reply by Fedora-Man** [reply]
Thank you for your assitance.

For no. 2 I found out that I can hide the title attribute by adding the following filter:

```
add_filter( 'nav_menu_link_attributes', 'remove_title_from_menu', 10, 4 );
function remove_title_from_menu( $atts, $item, $args, $depth ){
	unset($atts['title']);
	return $atts;
}
```
